pkg/oc/lib/newapp/app/pipeline.go

```diff
@@ -432,28 +431,6 @@ func addService(containers []corev1.Container, objectMeta metav1.ObjectMeta, sel
 	return svc
 }
 
-// AddRoutes sets up routes for the provided objects.
-func AddRoutes(objects Objects) Objects {
```

Comment:
This looks weird.  What are you doing instead?

Reply:
I didn't see where this was used. I reverted my removal.